ddtrace/constants.py

```diff
@@ -58,3 +58,110 @@
 AUTO_KEEP = 1
 # Use this to explicitly inform the backend that a trace should be kept and stored.
 USER_KEEP = 2
+
+APPSEC_BLOCKED_RESPONSE_HTML = """
```

Comment:
should this be minified to reduce overhead in the customer application?

Reply:
Good suggestion.